.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "3.1.1"
```

Comment:
```suggestion
  ".": "3.2.0"
```
I guess this is the current version of the project using release-please?

Reply:
version has set to 3.2.0